app/helpers/reservations_helper.rb

```diff
@@ -43,18 +43,16 @@ def manage_reservations_btn # rubocop:disable all
     return if (cannot? :manage, Reservation) || @reservation.reserver.id.nil?
     if (can? :override, :reservation_errors) &&
        @reservation.approval_status == 'requested'
-      link_to 'Review Request', review_request_path, class: 'btn btn-inverse'
+      link_to 'Review Request', review_request_path, class: 'btn btn-default'
     elsif @reservation.status == 'reserved'
-      link_to 'Check-Out',
-              manage_reservations_for_user_path(@reservation.reserver.id,
-                                                anchor: 'check_out_row'),
-              class: 'btn btn-inverse'
+      link_to 'Check-Out', manage_reservations_for_user_path(
+      @reservation.reserver.id,
```

Comment:
Why the new indentation? I'm pretty sure the old one was more stylistically correct (aligning parameters of each method).